cmd/v3tool/main.go

```diff
@@ -125,17 +143,23 @@ func run() int {
 			Timestamp:  time.Now().Unix(),
 		}
 
-		feeAddrResp, err := vClient.FeeAddress(context.TODO(), feeAddrReq, commitmentAddr)
+		feeAddrResp, err := vClient.FeeAddress(ctx, feeAddrReq, commitmentAddr)
 		if err != nil {
+			if errors.Is(err, context.Canceled) {
+				return 0
+			}
 			log.Errorf("getFeeAddress error: %v", err)
 			break
```

Comment:
I see most of these are `break`ing out to the `return 0` while others, along with the new code, are doing `return 0` directly.  It probably should be consistent.

Reply:
Updated